mantis-server/mantis-server-agent/src/main/java/io/mantisrx/server/agent/TaskExecutor.java

```diff
@@ -174,6 +175,11 @@ public TaskExecutor(
             Executors.newFixedThreadPool(
                 Hardware.getNumberCPUCores(),
```

Comment:
shall we do #cores - 1 now?

Reply:
the container can still get multiple cores?